src/font.ts

```diff
@@ -7,6 +7,15 @@ import { PetalumaMetrics } from './fonts/petaluma_metrics';
 import { CustomFont } from './fonts/custom_glyphs';
 import { CustomMetrics } from './fonts/custom_metrics';
 
+export interface FontStruct {
+  size: number;
+  weight: string;
+  family: string;
+  style?: string;
+  font?: string;
+  face?: string;
```

Comment:
`family`, `font`, and `face` are the same, also `point` and `size`. Merge them? or are they used differently?

Reply:
Done